frontend/lib/about/about_page.dart

```diff
@@ -117,9 +119,14 @@ class AboutPageState extends State<AboutPage> {
               ContentTile(icon: Icons.copyright, title: t.about.licenses(n: 1), children: getCopyrightText(context)),
               ListTile(
                 leading: const Icon(Icons.privacy_tip_outlined),
-                title: Text(t.about.privacyDeclaration),
+                title: Text(t.about.privacyPolicy),
                 onTap: () => launchUrlString(buildConfig.dataPrivacyPolicyUrl, mode: LaunchMode.externalApplication),
               ),
+              ListTile(
+                leading: const Icon(Icons.privacy_tip_outlined),
```

Comment:
```suggestion
                leading: const Icon(Icons.accessibility_new_outlined),
```
i think its nicer to use a separate icon